pkg/controller/controller_binding.go

```diff
@@ -25,6 +25,7 @@ import (
 	utilfeature "k8s.io/apiserver/pkg/util/feature"
 
 	"bytes"
```

Comment:
bytes should go up by net and fmt

Reply:
Wow, not sure why goimports didn't move that for me. It's usually really good at taking of that. I'll fix that.